src/Umbraco.Forms.Integrations.sln

```diff
@@ -35,7 +35,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Commerce", "Commerce", "{F4
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "eMerchantPay", "eMerchantPay", "{854D0DF1-AAEE-406A-82A8-B05992A0A74B}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Umbraco.Forms.Integrations.Commerce.EMerchantPay", "Umbraco.Forms.Integrations.Commerce.EMerchantPay\Umbraco.Forms.Integrations.Commerce.EMerchantPay.csproj", "{FD1A8505-A19E-4281-8701-6BA708C03CBE}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Umbraco.Forms.Integrations.Commerce.Emerchantpay", "Umbraco.Forms.Integrations.Commerce.EMerchantPay\Umbraco.Forms.Integrations.Commerce.Emerchantpay.csproj", "{FD1A8505-A19E-4281-8701-6BA708C03CBE}"
```

Comment:
Just for completeness, you could change the casing on this folder too: `Umbraco.Forms.Integrations.Commerce.EMerchantPay`.

If you do it using `git mv` it'll hopefully keep the history OK, but you might want to check.  I.e. merge this one, and then do the rename and create a new PR.  If GitHub sees it as a rename and not a tonne of deletes and additions, should be OK to merge.